datahub-upgrade/src/main/java/com/linkedin/datahub/upgrade/restoreindices/RestoreIndices.java

```diff
@@ -42,7 +41,6 @@ public String id() {
       final EntityRegistry entityRegistry, final EntitySearchService entitySearchService,
       final GraphService graphService) {
     final List<UpgradeStep> steps = new ArrayList<>();
-    steps.add(new GMSQualificationStep());
```

Comment:
We dont need this here?

Reply:
No. Since these don't depend on nocode, no reason to be there. Think I just copied and pasted without thinking too much about it